wiki/Hosting.md

```diff
@@ -0,0 +1 @@
+Under construction!
```

Comment:
This is out of sync with the wiki, someone already added content. I also disabled wiki edits from non maintainers. We should add a Not into the sidebar/footer on how the contents can be edited.